java/client/src/main/java/glide/api/commands/ListBaseCommands.java

```diff
@@ -67,6 +67,16 @@ public interface ListBaseCommands {
      */
     CompletableFuture<String[]> lpopCount(String key, long count);
 
+    /**
+     * Returns the length of the list stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/llen/">redis.io</a> for details.
+     * @param key The key of the list.
+     * @return The length of the list at <code>key</code>.<br>
+     *     If <code>key</code> does not exist, it is interpreted as an empty list and 0 is returned.
```

Comment:
```suggestion
     *     If <code>key</code> does not exist, it is interpreted as an empty list and <code>0</code> is returned.
```

Reply:
I will leave this as is to follow their documentation after I have finished all commands we can decide on what needs to be highlighted or not.